### PR TITLE
[dev v2.6] k8s november patch release 

### PR DESCRIPTION
**Pull request by vardhaman22**
created pr against a custom branch names dev-v2.6-1122-patches (created from dev-v2.6) because OOB releases.
for issue: https://github.com/rancher/rancher/issues/39370

### CODE REVIEW

File: rke/k8s_rke_system_images.go [hunk 1]
```diff
@@ -8522,5 +8522,137 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			IngressBackend:            m("k8s.gcr.io/defaultbackend:1.4"),
 			MetricsServer:             m("gcr.io/google_containers/metrics-server-amd64:v0.3.1"),
 		},
+		// Enabled out of band post v2.6.9
+		"v1.22.16-rancher1-1": {
```
Comment (snasovich) [inline rke/k8s_rke_system_images.go]: There probably won't be 1.22, see more info here: https://github.com/rancher/hyperkube/pull/291#issuecomment-1308896775

Comment (snasovich) [reply rke/k8s_rke_system_images.go]: Disregard, 1.22.16 actually got released (https://github.com/kubernetes/kubernetes/releases/tag/v1.22.16)

**Review by snasovich — CHANGES_REQUESTED** [review]
Same issue as https://github.com/rancher/kontainer-driver-metadata/pull/999#discussion_r1022126576

**Comment by vardhaman22** [comment]
> Same issue as [#999 (comment)](https://github.com/rancher/kontainer-driver-metadata/pull/999#discussion_r1022126576)

done

**Review by snasovich — APPROVED** [review]
LGTM!

**Review by kinarashah — APPROVED** [review]
- [x] only hyperkube image changes between v1.22.15 and v1.22.16 
- [x] only hyperkube image changes between v1.23.12 and v1.23.14 
- [x] only hyperkube image changes between v1.24.6 and v1.24.8 
- [x] version info and default version updated 
- [x] no service option changes 
- [x] no addon template changes

**Pull request merged**